modules/ROOT/pages/6.0-release-notes-ui-and-ux-elements-and-components.adoc

```diff
@@ -0,0 +1,263 @@
+
+In {productname} 6.0, editors without a `height` value specified in the {productname} configuration default to a 400px high text entry area.
+
+As was the case in {productname} 5.x, this new default only applies to editors in `iframe` mode, not `inline` mode.
+
+
+[[status-bar-compeny-id-changed-from-text-to-logo]]
+==== status bar company ID changed from text string to logo mark
+
+The _Powered by Tiny_ text string set in the {productname} status bar is now the {productname} logo.
+
+NOTE: The logo still responds appropriately when {productname} is used with a screen reader.
+
+
+[[status-bar-path-delimiter-character-changed]]
+==== status bar path delimiter character changed.
+
+The default status bar path delimiter character (also known as a breadcrumb separator) has been changed.
+
+It was `»` (the double, right-pointing, angled quotation mark).
+
+It is now `›` (the single, right-pointing, angled quotation mark).
+
+
+[[fixed-ui-and-ux-elements-and-components]]
+=== Fixed UI and UX elements and components
+
+
+[[anchor-elements-render-correctly]]
+==== Anchor elements now render correctly when `allow_html_in_named_anchor: true`
+
+When `allow_html_in_named_anchor` was set to `true`, content inside the anchor element was wrapped such that each character was set on a new line.
+
+With this release, setting `allow_html_in_named_anchor: true` sets characters inside the anchor element as entered, with no wrapping of each character to a new line.
+
+
+[[default-ui-theme-silver-renders-correctly]]
+==== Default UI theme, silver, no longer renders incorrectly as plugins load
+
+Previously, {productname}’s default UI theme, `silver`, was initialised such that it rendered some UI components before plugins that, potentially, would over-ride or adjust the defaults presented by `silver`.
+
+This resulted in these UI elements rendering incorrectly.
+
+With this release, `silver` no longer renders these UI elements before plugins get to over-ride or adjust, and these components no longer render incorrectly.
+
+
+[[dialog-labels-render-correcttly]]
+==== dialog labels now rendering correctly
+
+Previously, {productname} dialog labels and other UI elements were rendered with HTML markup.
+
+With this release, the `text` and `label` properties of these elements are now rendered as plain text, as expected.
+
+
+[[editor-selection-setring]]
+==== `editor.selection.setRng`
+
+Calls to `editor.selection.setRng` now update the insertion point bookmark when focus is returned to the editor.
+
+For the end-user, this change means content dragged from outside the {productname} editor into the editable area will be placed into the {productname} document where the insertion point appears as the drag is completed.
+
+This is the expected behavior.
+
+
+[[sub-menu-items-now-read-by-screen-readers]]
+==== sub-menu items now read by screen readers
+
+Previously, navigating through {productname} menus using the keyboard did not cause screen readers to read out menu items.
```

Comment:
This only happened on newer versions of Chrome so not sure if we want to clarify that?

Reply:
to quote my just sent commit message:

*sub-menu items now read by screen readers* section: text now explicitly notes the screen reader not working as expected only happened when the browser was Google Chrome.